cmd/newrelic-infra-service/newrelic-infra-service.go

```diff
@@ -1,5 +1,7 @@
 // Copyright 2020 New Relic Corporation. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
+//go:generate goversioninfo
```

Comment:
This will mark the binary with the version 1.0.0.0 
We need to use a similar mechanism as for newrelic-infra.go to replace the default version with the TAG.
If the correct version is not required, we can implement it on the new pipeline